futaba/delayed.py

```diff
@@ -40,20 +39,22 @@ def push(self, coro):
         self.queue.put_nowait(coro)
 
     async def main_loop(self):
-        while True:
+        for i in itertools.count():
```

Comment:
this feels like you'll run into issues once the number gets large. perhaps reset once it hits `delay_chunk_size` instead?

Reply:
Python is pretty good at supporting very large integers, I don't expect this to be a problem.